x/gov/keeper.go

```diff
@@ -131,6 +178,19 @@ func (keeper Keeper) getNewProposalID(ctx sdk.Context) (proposalID int64, err sd
 	return proposalID, nil
 }
 
+// Peeks the next available ProposalID without incrementing it
```

Comment:
this looks wrong because I clearly can see an increment. We should make it clear that this func mutates store.

Reply:
Oops, good catch.  Fixed.